.github/workflows/acceptance-tests-runner.yml

```diff
@@ -133,7 +137,7 @@ env:
   TF_LOG: ${{ vars.LOG_LEVEL }}
   ACCTEST_TIMEOUT: ${{ vars.ACCTEST_TIMEOUT }}
   # Only Migration tests are run when testing a specific previous provider version instead of the latest one
-  ACCTEST_REGEX_RUN: ${{ inputs.provider_version == '' && '^Test(Acc|Mig)' || '^TestMig' }}  
+  ACCTEST_REGEX_RUN: ${{ inputs.provider_version == '' && '^TestAccNetworkRSNetworkPeering_basicAzure' || '^TestMig' }}  
```

Comment:
looks like this was a temporary change?

Reply:
fixed, resolving comment, feel free to reopen